.github/workflows/go.yml

```
@@ -20,7 +25,7 @@ jobs:
     - name: Set up Go 1.x
       uses: actions/setup-go@v2
```

actionlint (1.7.7)

the runner of "actions/setup-go@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)